contexts/VotesContext.tsx

```diff
@@ -184,11 +201,18 @@ export function VotesProvider({ children }: { children: ReactNode }) {
         encryptedVote: encryptedVotes[uniqueKey],
         decryptedVote: decryptedVotes[uniqueKey],
         contentfulData: contentfulData[uniqueKey],
-        transactionHash: voteTransactionHashes[uniqueKey],
+        transactionHash: transactionHashes[uniqueKey],
+        voteHistory: voteHistoryByKey[uniqueKey] ?? {
+          uniqueKey,
```

Comment:
We use empty default data so that we can still show a chart of (empty) values in the vote history panel